openlibrary/plugins/upstream/utils.py

```diff
@@ -710,51 +710,58 @@ def strip_accents(s: str) -> str:
 
 
 @functools.cache
-def get_languages() -> dict:
-    keys = web.ctx.site.things({"type": "/type/language", "limit": 1000})
+def get_languages(limit: int = 1000) -> dict:
+    keys = web.ctx.site.things({"type": "/type/language", "limit": limit})
     return {lang.key: lang for lang in web.ctx.site.get_many(keys)}
 
 
+def _normalize(s: str) -> str:
+    return strip_accents(s).lower()
+
+
+def _matches_lang_name(
+    word_prefix: str, language: dict, prefix_language: str | None = None
+) -> Storage | None:
+    lang_name = language.name
```

Comment:
`language` is a dict...
```suggestion
    lang_name = language["name"]
```